tests/settings.py

```diff
@@ -0,0 +1,104 @@
+"""
+Django settings for running tests for Resolwe package.
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+
+PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+SECRET_KEY = 'secret'
+
+DEBUG = True
+
+MIDDLEWARE_CLASSES = (
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+)
+
+
+INSTALLED_APPS = (
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.staticfiles',
+
+    'rest_framework',
+    'guardian',
+    'mathfilters',
+    'versionfield',
+
+    'resolwe',
+    'resolwe.apps',
```

Comment:
Resolwe-bio is package, so apps functionalities will be needed to register processors.